### PR TITLE
fix: safe negative-RT handling and JAX backend for RDM3

### PR DESCRIPTION
**Pull request by krishnbera**
- Replace element-wise jax_check_parameters for rt > 0 with jnp.where + LOGP_LB floor in logp_rdm3. This clamps negative decision times to a small epsilon before passing to the PDF/CDF, preventing NaN in the likelihood and providing a finite gradient floor per trial.
- Set backend to "jax" and add default HalfNormal priors for A and t in both racing_diffusion_3_config and rdm3_config.

Made-with: Cursor

### CODE REVIEW

File: src/hssm/likelihoods/analytical.py [hunk 1]
```diff
@@ -497,9 +498,13 @@ def logp_rdm3(
     response = data_reshaped[:, 1]
     response_int = response.astype(jnp.int_)
 
-    logp = _jax_rdm3_ll(rt, response_int, A, b, v0, v1, v2).squeeze()
+    negative_rt = rt <= 0.0
```
Comment (cpaniaguam) [inline src/hssm/likelihoods/analytical.py]: May I suggest the name  `is_negative_rt`?

**Review by cpaniaguam — COMMENTED** [review]
lgtm. Just one little suggestion.
